wiki/Tournaments/MCNC/2021/en.md

```diff
@@ -0,0 +1,130 @@
+---
+tags:
+  - MCNC
+  - MCNC2021
+---
+
+# osu!mania 4K Chinese National Cup 2021
+
+![MCNC 2021 logo](img/logo.png)
+
+The **osu!mania 4K Chinese National Cup 2021** (***MCNC 2021***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the third installment of the osu!mania 4K Chinese National Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2021-03-01/2021-03-31 |
+| Qualifying Contest | 2021-04-05/2021-04-11 |
```

Comment:
should be `Qualifiers` (instead of `Qualifying Contest`) i suppose since in the Mappools section below you have already written in `Qualifiers`

Reply:
Done!